app/models.py

```diff
@@ -354,22 +358,37 @@ class Config:
 class UpdateSession(BaseModel):
     """Model for the body of the request that updates a session"""
 
-    has_quiz_ended: bool
+    has_quiz_ended_first_time: bool
```

Comment:
Ended is only for last time. Maybe rename to `has_quiz_stopped_first_time`?

Reply:
yes